redbot/cogs/mutes/mutes.py

```diff
@@ -1467,7 +1477,78 @@ async def unmute(
                         user, author, guild, _("Server unmute"), reason
                     )
                 else:
-                    issue_list.append(success)
+                    issue_list.append(response)
+        self._channel_mute_events[guild.id].set()
+        if success_list:
+            if ctx.guild.id in self._server_mutes and self._server_mutes[ctx.guild.id]:
+                await self.config.guild(ctx.guild).muted_users.set(
+                    self._server_mutes[ctx.guild.id]
+                )
+            else:
+                await self.config.guild(ctx.guild).muted_users.clear()
+            await ctx.send(
+                _("{users} unmuted in this server.").format(
+                    users=humanize_list([f"{u}" for u in success_list])
+                )
+            )
+        if issue_list:
+            await self.handle_issues(ctx, issue_list)
+
+    @commands.command(usage="<users...> [reason]", hidden=True)
+    @commands.guild_only()
+    @commands.mod_or_permissions(manage_roles=True)
+    async def forceunmute(
+        self,
+        ctx: commands.Context,
+        users: commands.Greedy[discord.Member],
+        *,
+        reason: Optional[str] = None,
+    ):
+        """Force Unmute users who have had channel overwrite mutes in every channel.
+
+        `<users...>` is a space separated list of usernames, ID's, or mentions.
+        `[reason]` is the reason for the unmute.
+        """
+        if not users:
+            return await ctx.send_help()
+        if ctx.me in users:
+            return await ctx.send(_("You cannot unmute me."))
+        if ctx.author in users:
+            return await ctx.send(_("You cannot unmute yourself."))
+        async with ctx.typing():
+            guild = ctx.guild
+            author = ctx.author
+            issue_list = []
```

Comment:
Nitpick but `[p]unmute`/`[p]channelunmute` sets `audit_reason` generated from `reason` and passes that to `channel_unmute_user()` instead:
```suggestion
            audit_reason = get_audit_reason(author, reason, shorten=True)
            issue_list = []
```